src/main/resources/templates/authenticator/freja-eid/authenticate/error.vm

```diff
@@ -0,0 +1,30 @@
+#*
+*  Copyright 2018 Curity AB
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*#
+
+#define ($_body)
+
+<div class="mt3 px3 lg-px4" id="user-error">
+    <div class="alert alert-danger px4">
+        #message("authenticator.freja-eid.authenticate.${error}")<br>
+    </div>
+</div>
+<div class="center py2 login-actions">
+    <a href="$_authUrl" role="button">
+        <i class="icon ion-refresh inlineicon"></i>#message("authenticator.freja-eid.authenticate.error.restart")</a>
+</div>
+#end
+
+#parse("layouts/default")
```

Comment:
they error layout shouldn't be used

Reply:
It is a known error, when the user has declined the login in the app, or when the authentication has expired etc. I want the users to see the error and be able to restart, so this is the correct choice.